src/main/java/edu/pwr/pizzeria/service/order/CustomerOrderException.java

```diff
@@ -0,0 +1,8 @@
+package edu.pwr.pizzeria.service.order;
+
+public class CustomerOrderException extends RuntimeException{
```

Comment:
I would suggest changing the name to be more specific

Reply:
CustomerOrderNotFoundException?